spacy/tests/pipeline/test_entity_linker.py

```diff
@@ -807,6 +809,107 @@ def create_kb(vocab):
     assert_equal(batch_deps_1, batch_deps_2)
     assert_equal(batch_deps_1, no_batch_deps)
 
+    eval = nlp.evaluate(train_examples)
+    assert "nel_macro_p" in eval
+    assert "nel_macro_r" in eval
+    assert "nel_macro_f" in eval
+    assert "nel_micro_p" in eval
+    assert "nel_micro_r" in eval
+    assert "nel_micro_f" in eval
+    assert "nel_f_per_type" in eval
```

Comment:
Formatting in suggestion TBD. Ignore if you separated on purpose for easier debugging.
```suggestion
    assert all([metric in eval for metric in ("nel_macro_p", "nel_macro_r", "nel_macro_f", "nel_micro_p", "nel_micro_r", "nel_micro_f", "nel_f_per_type")]
```

Reply:
In unit tests I prefer to test one thing per line so it's clear what fails when it does ;-)